### DISCUSSION TITLE
Monitoring configuration for Windows / Linux is cumbersome

### DISCUSSION THREAD

**Discussion by pengjiangling** [ccfos/nightingale#1408] [Q&A]
事情：我有很多台主机，我需要对这些主机进行监控以及分类，分类这里就用到了categraf的标签Label功能，但是监控这些主机，我需要去开启cpu、disk、diskio、kernel、linux_sysctl_fs、mem、net、netstat、processes、system，并且这些插件的每一个toml配置都要加相同的标签（就比如说我加标签为{ip = 10.1.1.1}，那么这些所有的插件都要设置一遍标签，而且一样因为他们来自于同一台主机），这里能不能优化，合并一下

**Comment by wushengwusheng** [comment]
categraf里有个global_label 应该可以解决你的问题

**Reply by pengjiangling** [reply]
谢谢，解决了
